changelog/fix_rails_save_bang_to_handle_instance_variables

```diff
@@ -0,0 +1 @@
+* [#1228](https://github.com/rubocop/rubocop-rails/issues/1228): Enhance `Rails/SaveBang` to properly handle instance variables. ([@ydakuka][])
```

Comment:
This file is missing the `.md` extension and I think it will not be picked up by the changelog generator because of that

Reply:
Oops, fixed.